### PR TITLE
Fix issue with none well formated  "Accept" headers.

### PR DESCRIPTION
**Pull request by SamHecquet**
Add security to prevent none well formated param into "Accept" headers.

### CODE REVIEW

File: library/Zend/Http/Header/AbstractAccept.php [hunk 1]
```diff
@@ -167,6 +167,11 @@ protected function getParametersFromFieldValuePart($fieldValuePart)
             }
 
             foreach ($paramsStrings as $param) {
+                // Prevents none well formated param
+                if (strstr($param, '=') === false) {
```
Comment (Martin-P) [inline library/Zend/Http/Header/AbstractAccept.php]: http://php.net/strstr
> Note:
If you only want to determine if a particular needle occurs within haystack, use the faster and less memory intensive function strpos() instead.

**Comment by Ocramius** [comment]
Requires test case

**Comment by weierophinney** [comment]
Ping @SamHecquet — we need a test case to accept this.

**Comment by Maks3w** [comment]
Duplicate https://github.com/zendframework/zend-http/pull/25

**Pull request closed without merging**